pkg/ebpf/ebpftest/buildmode_windows.go

```diff
@@ -0,0 +1,35 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
```

Comment:
windows build tag?

Reply:
Unnecessary, because of Go naming conventions.

Reply:
> If a file's name, after stripping the extension and a possible _test suffix, matches any of the following patterns:
> `*_GOOS`
> `*_GOARCH`
> `*_GOOS_GOARCH`
> (example: source_windows_amd64.go) where GOOS and GOARCH represent any known operating system and architecture values respectively, then the file is considered to have an implicit build constraint requiring those terms (in addition to any explicit constraints in the file).

Reply:
super useful tip, thanks